resources/puppetlabs/lein-ezbake/template/global/ext/redhat/ezbake.spec.erb

```diff
@@ -93,6 +111,11 @@ Requires:         chkconfig
 Requires:         aaa_base
 %endif
 
+<% if !EZBake::Config[:is_pe_build] -%>
+# Required to get trustanchors for java
+BuildRequires: %{_certs_package}
```

Comment:
but, we should definitely maintain what was previously set here, so I'm not asking for a change here